packages/suite/src/views/guide/Feedback.tsx

```diff
@@ -274,7 +276,15 @@ const Feedback = ({ type }: Props) => {
                     </CharacterCount>
                 </TextareaWrapper>
 
-                <Submit onClick={onSubmit}>
+                <Submit
+                    onClick={() => {
+                        onSubmit();
+                        analytics.report({
+                            type: 'guide/feedback/submit',
+                            payload: { type: type.toLowerCase() as 'bug' | 'suggestion' },
```

Comment:
nit:
I would prefer 
```javascript
payload: { type: type.toLowerCase() === 'BUG' ? 'bug' : 'suggestion' },
```
or something like that to prevent use of unsafe `as` param.

Another thing is, that would be nice to use constants or enum items instead of regular strings for types (as `'BUG'` | `'SUGGESTION'` or `guide/feedback/submit`) but it's something for general refactoring, definitely not in this PR, I know.

Reply:
Resolved. Agree that we should start using constants or enums.